package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "postinstall": "lerna bootstrap --hoist",
-    "updateApis": "lerna run updateApis",
+    "updateApis": "lerna run updateApis --stream",
```

Comment:
Why this change here?

Reply:
lerna doesn't show logs from the child processes by default, it only shows the end result